ElectronClient/InteropServiceHelper.js

```diff
@@ -92,6 +95,17 @@ class InteropServiceHelper {
 		return this.exportNoteTo_('printer', noteId, options);
 	}
 
+	static async defaultFilename(noteIds, fileExtension) {
+		const note = await Note.load(noteIds[0]);
+		const folder = await Folder.load(note.parent_id);
+
+		const foldername = friendlySafeFilename(folder.title, 100);
+		const filename = friendlySafeFilename(note.title, 100);
+
+		// friendlySafeFilename assumes that the file extension is added after
+		return `${foldername}-${filename}.${fileExtension}`;
```

Comment:
Could you add a space before and after the dash please? `${foldername} - ${filename}.${fileExtension}`